metpy/calc/thermo.py

```diff
@@ -5,6 +5,7 @@
 
 from __future__ import division
 
+from math import isclose
```

Comment:
python 3.5+ here :)

Reply:
Well that's good because our docs say:
> We support Python >= 3.6 and currently support Python 2.7.

😁

Reply:
Oh, Python 2.7. DOH!

Reply:
My world will be so much better when Python 2.7 is just a distant memory.